packages/core/src/components/ui/Dropdown/Dropdown.List.js

```diff
@@ -50,7 +63,9 @@ const DropdownList = (props) => {
               value={searchTerm}
               onChange={handleSearchTermClick}
             />
-            <FontAwesomeIcon className='search-icon' icon={faSearch} />
+            {searchTerm
+              ? <FontAwesomeIcon icon={faTimes} className='search-icon' onClick={onCancelClick} />
```

Comment:
this onClick wont be accessible with keyboard, so can you add <Button onClick=... /> wrapper, similar to https://github.com/bitfinexcom/ufx-ui/blob/staging/packages/core/src/components/TickerList/TickerList.Toolbar.js#L22